### PR TITLE
Bump module version in usage example

### PR DESCRIPTION
**Pull request by kamilhristov**
# PR o'clock

## Description

Copy/pasting the example is not working out of the box. It might take some time to notice the version difference.

### Checklist

- [x] `terraform fmt` and `terraform validate` both work from the root and `examples/eks_test_fixture` directories (look in CI for an example)
- [x] Tests for the changes have been added and passing (for bug fixes/features)
- [x] Test results are pasted in this PR (in lieu of CI)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Any breaking changes are noted in the description above

### CODE REVIEW

**Comment by brandoconnor** [comment]
Can you change this to omit the version completely? Needing to remember to bump the example each time is going to be a persistent burden. I thought someone else did this already but I guess not. IIRC it defaults to the latest release when no version is specified.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ A full example leveraging other community modules is contained in the [examples/
 ```hcl
 module "eks" {
   source                = "terraform-aws-modules/eks/aws"
-  version               = "0.1.0"
+  version               = "1.1.0"
```
Comment (brandoconnor) [inline README.md]: kill this line with fire.

**Comment by kamilhristov** [comment]
@brandoconnor comment addressed. I was thinking of removing it completely. 😄  It is better to use _latest_ for testing purposes.

**Comment by brandoconnor** [comment]
Thanks for this!

**Pull request merged**